jxls-site/src/site/markdown/reference/reader.md.vm

```diff
@@ -29,23 +29,23 @@ Construction of XLSReader using XML config file
 To use Jxls-Reader to parse excel files and populate your Java objects with read data you have to construct *XLSReader* object first. 
 The easiest way to do this is to use a special XML configuration file. 
 
-Let's use [departmentdata.xls](../xls/departmentdata.xls) Excel file with some department data to demonstrate this method
+Let's use [data.xls](../xls/data.xls) Excel file with some  data to demonstrate this method
```

Comment:
Error?! Why renaming departmentdata.xls to data.xls? departmentdata.xls exists, data.xls doesn't.